Makefile

```diff
@@ -58,6 +58,13 @@ examples_install:
 	${MKDIR} ${SCRIPTS_INSTALL_PATH}/examples/dnsdoctor
 	${INSTALL} -m 0644 examples/dnsdoctor/*.lua ${SCRIPTS_INSTALL_PATH}/examples/dnsdoctor
 
+ipparse_install:
+	wget https://github.com/luainkernel/moontastik/releases/latest/download/moontastik_lua.zip -O moontastik_lua.zip
+	[ -d moontastik_lua ] && ${RM} -r moontastik_lua || true
+	unzip moontastik_lua.zip
+	${MKDIR} ${SCRIPTS_INSTALL_PATH}/ipparse
+	cd moontastik_lua/ipparse && find . -type f -exec install -vDm 644 {} ${SCRIPTS_INSTALL_PATH}/ipparse/{} \; ; cd -
```

Comment:
I would move this to an "install" script in the `moonstatik` package

Reply:
I don’t understand this one: the whole point of this integration is to avoid MoonScript dependency. If I put this in `moontastik` (which doesn’t include the lua files), it will necessarily add steps to install `ipparse`.

Reply:
decoupling as well.. we can have `moonstatik/install.sh` for instance, and let the lunatik script just call it here.. lunatik doesn't need to know the inner details..